common/djangoapps/edxmako/paths.py

```diff
@@ -49,15 +54,23 @@ def add_directory(self, directory, prepend=False):
 
     def get_template(self, uri):
         """
-        Overridden method which will hand-off the template lookup to the microsite subsystem
+        Overridden method do template lookup database or in white labe site theme.
```

Comment:
"Overridden method for locating a template in either the database or the site theme."